public/js/addTask.js

```diff
@@ -1,4 +1,4 @@
-async function taskFormHandler(event) {
+async function newTaskFormHandler(event) {
```

Comment:
As I mentioned above, I think the way we have been writing this in class is:
const newTaskFormHandler = async (event) => {